packages/api-explorer/src/lib/get-auth.js

```diff
@@ -32,24 +32,30 @@ function getSingle(user, scheme = {}, selectedApp = false) {
   return getKey(user, scheme);
 }
 
-function getAuth(user, oasFiles) {
+function getAuth(user, oasFiles, selectedApp = false) {
   return Object.keys(oasFiles)
     .map(id => {
       const oas = oasFiles[id];
 
       if (
         Object.keys(oas.components || {}).length === 0 ||
         Object.keys(oas.components.securitySchemes || {}).length === 0
-      )
+      ) {
         return {};
+      }
 
       return Object.keys(oas.components.securitySchemes)
         .map(scheme => {
           return {
-            [scheme]: getSingle(user, {
-              ...oas.components.securitySchemes[scheme],
-              _key: scheme,
-            }),
+            [scheme]: getSingle(
+              user,
+              {
+                ...oas.components.securitySchemes[scheme],
+                _key: scheme,
+                selectedApp,
```

Comment:
Is this needed?

Reply:
Yeah, `getSingle` needs to know the current app so it can pick out the right key.